### DISCUSSION TITLE
A toy configuration language and data format

### DISCUSSION THREAD

**Discussion by ThatXliner** [lark-parser/lark#884] [Show and tell]
ModCFG, a whitespace-sensitive configuration language and data format that supports enums, datetimes, and other stuff. (It's a *toy language* because there's already so many others out there.)

This really proves how powerful (and easy to use) Lark is.

<sub>My biggest achievement is that I implemented most functionality in 1 commit</sub>

[Repo](https://github.com/ThatXliner/modcfg)
[PyPi](pypi.org/project/modcfg/)

**Comment by erezsh** [comment]
Looks nice!

**Reply by erezsh** [reply]
I bet if you wrote a blog post detailing how you wrote that, it would get a decent amount of views.
